index.php

```diff
@@ -7768,10 +7769,33 @@ function displayLookupResults(res, userSelection) {
 			res.forEach((comp) => {
 				// let compHtml = $(`<p class="singleResult" style="display: none; font-weight:bold;">${comp}</p><hr style="display: none">`); // bold
 				// let compHtml = $(`<p class="singleResult" style="display: none; color: #4b636e;">${comp}</p><hr style="display: none">`); // grey
-				let compHtml = $(`<p class="singleResult" style="display: none; color: #112e51;">${comp}</p><hr style="display: none">`); // navy blue
-				$("#msaResultsList").append(compHtml);
-				$(compHtml).slideDown();
-			})
+				
+				// check if comp is in list of suppressed counties
+				let isSuppressed = false;
+				$.getJSON('suppressed-counties.json', function(json) { // add error-handling when possible
+
+					for (let i = 0; i < json.length; i++) {
```

Comment:
nit: is there a reason it's not a forEach loop (which is slightly more readable)? it might be good to also do something like `let i = 0; in= json.length; i < n; i++` so you're only retrieving the json.length once. i mean, this is a really big nitpick, not a big deal

Reply:
leaving in due to ability to break early from loop

Reply:
changed to use .every() instead